### PR TITLE
Add security.txt

### PR DESCRIPTION
**Pull request by oehm-smith**
## Types of changes
- [  ] Bug fix (non-breaking change which fixes an issue)
- [ x ] New feature (non-breaking change which adds functionality)
- [  ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ NA] My code follows the code style of this project.  (No code changes made)
- [ x ] My change requires a change to the documentation.
- [ x ] I have updated the documentation accordingly.
- [ x ] I have read the **CONTRIBUTING** document.
- [ NA ] I have added tests to cover my changes.
- [ NA ] All new and existing tests passed.  (No code changes made)

### CODE REVIEW

**Comment by coliff** [comment]
Thanks for the PR - It's an interesting addition. I hadn't previously heard of this. it doesn't appear to be very well known (pun not intended) yet.

If this PR were to be merged, the additions to the `/dist/` would also need to be made in the `/src/` dir - otherwise they'll get rewritten when a new build is made. Also I think additional info is needed on that page to let users know that they need to edit the `security.txt` file before being used - the file there is just a starter template.

**Comment by roblarsen** [comment]
I see a lot of 404s for this `well-known` location. I think this would be a good educational addition. I don't know if the file itself should be there or it should just be documented. But anything that solves a persistent 404 is good in my book.

**Comment by oehm-smith** [comment]
PR updated with feedback.

File: dist/doc/usage.md [hunk 1]
```diff
@@ -121,6 +123,16 @@ technology powering it.
 
 Edit this file to include any pages you need hidden from search engines.
 
+### .well-known
+
+RFC5785 [https://tools.ietf.org/html/rfc5785](https://tools.ietf.org/html/rfc5785) defines '.well-known' as a unique location for content discovery.  It contains one file - security.txt.
+
+### security.txt
+
+When security risks in web services are discovered by independent security researchers who understand the severity of the risk, they often lack the channels to disclose them properly. As a result, security issues may be left unreported. Security.txt defines a standard to help organizations define the process for security researchers to disclose security vulnerabilities securely.
```
Comment (coliff) [inline dist/doc/usage.md]: > When security risks in web services are discovered by independent security researchers

security issues could be discovered by any user, not just security researchers so I suggest this line be changed to remove the 'by independent security researchers'.

**Comment by coliff** [comment]
Thanks again for this PR @oehm-smith - we've decided to give this a mention in the 'extend' docs with a link to https://securitytxt.org/ for more details.
Will be part of the #2074 PR

**Comment by oehm-smith** [comment]
Cool no worries Christian.  Apologies for not getting back to it as 
always became a lower priority.

**Pull request closed without merging**